### DISCUSSION TITLE
Cohesive Understandings as Optional Roads by Players Themselves

### DISCUSSION THREAD

**Discussion by ghost** [ValveSoftware/Dota2-Gameplay#20132] [Ideas]
LINK: https://github.com/crystalnovalce/MyDota_Words/discussions/8

Sorry if I shared it in the wrong way. 
I am not very professional in using GitHub.

**Comment by commentator24** [comment]
so basically what you want is a new UI for Play Dota side menu - that separates solo and party queue is it?
while i agree matchmaking should be improved, e.g. strict solo matchmaking is in settings not on play dota menu (players might not realise this) here are a few reasons your designs also flawed and backwards in design:

-the system should already understand if you're partied with someone then you can't solo queue, and if you solo you cant party queue. your design just adds unnecessary UI and coding to prevent wrong queue solo and party which are basically the same lobby.
-there is already a setting for strict solo queue for players who prefer solo. this allows freedom for players who do not care and just want to find a game within 10 minutes. but you might not find game for a long time. how do you solve the problem queuing strictly party queue if you cannot find match? dota solution is to mix them together vs solo queue within 10 min which seems good to me.
-solo queue does allow the formation of 2+2+1. but other team not necessarily have to be also 2+2+1. this is because it is shuffled for balance. e.g. originally it is 2 vs 2?
as shown here: 
![image](https://github.com/user-attachments/assets/58802b5b-393b-479d-b6e4-1c97c26e4356)
-im guessing 4+1 is not allowed because it creates a bias / toxic environment. e.g. 4 ppl report on 1 person not in team. how is this fair?
-5 usually vs 5. but sometimes 5 can vs 2+3 if there are no ppl.

if anything, i prefer to see UI for winning streaks before queuing next match and the value of MMR gained before accepting match. e.g. 
a window that shows This match worth 25+1+2 MMR etc for a more enjoyable experience. and extra 1, 2,3 mmr should make it obvious how many ppl are on win streaks e.g. if enemy on 3 win/lose streaks then this game is 25+3 mmr for you and indication who is the last match MVPs e.g. radiant has 2 MVP and 1 of them has 4 win streak, and dire has 3 MVP 2 of them has 3 win streak, so the value of the mmr should be like 1+4(5) vs 1+3+3(7) so its +30 mmr for radiant and +32 for dire. This way it would be more fun to try get win streaks and more mmr

**Reply by ghost** [reply]
### How the New Design of Matchmaking should look like?
4. Solo PLAY DOTA (no composition/formation of 2+2+1, 4+1, 3+1+1, or 5 for each team, and only allow 1+1+1+1+1)
- 4.1 Ranked MM (all pick or random draft)
- 4.2 Unranked MM (all-pick, turbo, single draft, random draft, ability draft)
5. Party PLAY DOTA (only allow composition/formation of 3+2, or 5 for each team)
- 5.1 Ranked MM (all pick or random draft)
- 5.2 Unranked MM (all-pick, turbo, single draft, random draft, ability draft)

Do you mean the "flaw" that might happen is the abuse of matchmaking which used to happen years ago when the players were using the "Captain Mode" to match themselves up through the 5v5 or 5v(3+2) or (3+2)v(2+3)? 

Could that be the reason why the "Captain Mode" was removed from the matchmaking? 

If the reasoning I said above is true, then let it be. I can more understand this topic now and will try to think better. May close the topic if necessary. Sorry for the bad reply as recently (10th/Oct) I suffered from a dark fate in my life, epilepsy(fit), and was deprived a lot (not a mental disorder, but my English dropped a lot). I was afraid my critical thinking might drop much too. :( 

Thanks for your reply and understanding.

Regards.

**Comment by Billy-39** [comment]
Players can choose different paths, like befriending [Krobus Stardew Valley](https://starvalleyapk.com/krobus-stardew-valley-guide/), to shape their unique journey.
